android/src/com/frostwire/android/gui/adapters/menu/RenameFileMenuAction.java

```diff
@@ -73,7 +83,9 @@ private void renameFile(String newFileName) {
         if (isValidFileName(newFileName)) {
             fd.filePath = Librarian.instance().renameFile(fd, newFileName);
             fd.title = FilenameUtils.getBaseName(newFileName);
-            adapter.notifyDataSetChanged();
+            if (adapter != null) {
```

Comment:
not an error, simply the action is being used in a context where there's no adapter to work with. wonder how to go about it, just trying to reuse the action. Perhaps have the adapter implement some sort of interface that gets notified, still the checking for a listener would exist, but it'd be more uniform.